.config/fish/config.fish

```diff
@@ -1,33 +1,3 @@
 if status is-interactive
     # Commands to run in interactive sessions can go here
 end
-
```

Comment:
Bring all this back...probably got deleted accidentally with all the symlinking shenanigans